dev/io.openliberty.jcache.internal_fat.hazelcast/fat/src/io/openliberty/jcache/internal/fat/JCacheAuthenticationCacheServerRestartTest.java

```diff
@@ -0,0 +1,128 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.jcache.internal.fat;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.ws.webcontainer.security.test.servlets.BasicAuthClient;
+import com.ibm.ws.webcontainer.security.test.servlets.ServletClient;
+
+import componenttest.annotation.CheckForLeakedPasswords;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * Contains distributed JCache authentication cache tests for server restart.
+ */
+@SuppressWarnings("restriction")
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
+public class JCacheAuthenticationCacheServerRestartTest extends BaseTestCase {
+
+    private static BasicAuthClient basicAuthClient1;
+    private static final int TTL_SECONDS = 10;
```

Comment:
Delete this, as we don't need TTL testing here (and I have disabled elsewhere b/c it really just tests the provider and not our code.
```suggestion
```